relprocotron/__main__.py

```diff
@@ -105,21 +105,23 @@ def main(
     click.echo(f"Output File: {output_file}")
 
     # Generate release activities JSON
+    _generate_release_activities(
+        release_name=release_name,
+        release_tag=release_tag,
+        release_type=release_type,
+        release_date=release_date,
+        project_url=project_url,
+        software_name=software_name,
+        software_version=software_version,
+        comments=comment_list,
+        output_file=output_file,
+        dry_run=dry_run
+    )
+
     if not dry_run:
```

Comment:
Remove this if, and just write out the file in all cases. Disregard the dry-run switch for now.